ssc/cardano-sl-ssc.cabal

```diff
@@ -16,33 +16,6 @@ library
   exposed-modules:
     Pos.Ssc
 
```

Comment:
Should this have disappeared? I don't see the `Pos.Ssc` module, and the only package that depends on `cardano-sl-ssc` is `cardano-sl-ssc-test`, which is also an empty package.

Reply:
Yes it should have.